tests/testthat/test-plot_lifeSat.R

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Good tests!

Reply:
Good that you spot this! I changed the function a few days ago and forgot to adapt the test. Thanks for pointing this out! :)